src/openms/source/ANALYSIS/QUANTITATION/IsobaricChannelExtractor.cpp

```diff
@@ -213,9 +215,10 @@ namespace OpenMS
     interpolate_precursor_purity_ = getParameters().getValue("purity_interpolation") == "true";
 
     /* check for sensible parameters */
-    if (dynamic_cast<const TMTTenPlexQuantitationMethod*>(quant_method_) != NULL && reporter_mass_shift_ > TMT_10PLEX_CHANNEL_TOLERANCE)
+    if (((dynamic_cast<const TMTTenPlexQuantitationMethod*>(quant_method_) != NULL) || (dynamic_cast<const TMTElevenPlexQuantitationMethod*>(quant_method_) != NULL))
```

Comment:
if dynamic casts for type checks are needed it is usually a sign of bad design (at least if you have control over the all types as we do)